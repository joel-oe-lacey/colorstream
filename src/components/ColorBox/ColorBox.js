import React from 'react';

const ColorBox = ({ props: {hex}}) => {
    return (
        <section></section>
    )
}

export default ColorBox;