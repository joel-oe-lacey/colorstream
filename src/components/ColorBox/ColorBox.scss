.color-cont {
    height: 100%;
    width: 100%;
    display: flex;
    flex-direction: column;
    justify-content: center;
    align-items: center;
    box-shadow: 9px 9px 16px rgba(163, 177, 198, 0.6), -9px -9px 16px  rgba(255,255,255, 0.5);
    border-radius: 10%;
}

.color-cont-inner {
    height: 90%;
    width: 90%;
    border-radius: 10%;
}