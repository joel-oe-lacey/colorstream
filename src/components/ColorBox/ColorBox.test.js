import React from 'react';
import { shallow } from 'enzyme';
import ColorBox from './ColorBox';

describe('ColorBox', () => {
    let wrapper;

    beforeEach(() => {
        wrapper = shallow(<ColorBox />);
    });

    it('should match the snapshot', () => {
        expect(wrapper).toMatchSnapshot();
    })
})