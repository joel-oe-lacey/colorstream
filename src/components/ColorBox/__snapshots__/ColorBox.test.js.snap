// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ColorBox should match the snapshot 1`] = `
<section
  className="color-cont"
>
  <div
    className="color-cont-inner"
    style={
      Object {
        "backgroundColor": "undefined",
      }
    }
  >
    <h3
      style={
        Object {
          "color": "#000000",
        }
      }
    >
      undefined
    </h3>
  </div>
</section>
`;
